### ISSUE TITLE
name 'read_crystfel' is not defined, on version 0.9.5

### ISSUE THREAD

**Issue by xvalad** [Hekstra-Lab/reciprocalspaceship#56]
Input:
```
dataset = read_crystfel("example.stream")
import reciprocalspaceship as rs
```

Output:
```
---------------------------------------------------------------------------
AttributeError                            Traceback (most recent call last)
<ipython-input-39-1212d53388f4> in <module>
----> 1 dataset = rs.read_crystfel("example.stream")

AttributeError: module 'reciprocalspaceship' has no attribute 'read_crystfel'
```

**Comment by JBGreisman** [comment]
Thanks for raising this -- `rs.read_crystfel()` was added in #50, but I haven't yet made a new release. I will push a new release (0.9.6) with this function to PyPI today. 

In the meantime, you can get this functionality by cloning the repo and installing from the local copy.

**Comment by JBGreisman** [comment]
Just released `rs 0.9.6` --  this should be resolved now in the current PyPI version that gets installed via `pip`. I'm closing the issue, but please reopen if you run into more issues.
